Polish/przygoda/jedzenie/akap/akap.md

```diff
@@ -0,0 +1,21 @@
+Siedzisz właśnie w swojej piwnicy, która od niepamiętnych czasów jest twoim domem. 
+Mieszkasz tu odkąd twoi rodzice wyrzucili cię z domu, gdy dowiedzieli się ze jesteś libertarianinem. 
+Siedzisz na stołku i z opartymi łokciami o biurko spoglądasz na monitor swojego komputera. 
+Lurkujesz właśnie swoją ulubioną grupę na Facebooku "Jak będzie w akapie" chichocząc się pod nosem ze śmieszkowych postów
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.